### PR TITLE
Misc fixes

### PR DESCRIPTION
**Pull request by graydon**
Just a few improvements from the past couple day's fiddling with benchmarking runs.

### CODE REVIEW

**Comment by graydon** [comment]
@marta-lokhova do these look ok to you? (don't know why, this repo doesn't seem to support review-requests)

**Comment by marta-lokhova** [comment]
Looks good, thank you! I don't have write access to this repo for some reason, @MonsieurNicolas can you merge this?

File: lib/stellar_core_commander/local_process.rb [hunk 1]
```diff
@@ -166,6 +166,7 @@ def config
         INVARIANT_CHECKS=#{invariants.to_s}
 
         [QUORUM_SET]
+        THRESHOLD_PERCENT=51
```
Comment (MonsieurNicolas) [inline lib/stellar_core_commander/local_process.rb]: why do we need this smaller number than the default? It it that in other places we use 51 as threshold?

Comment (marta-lokhova) [reply lib/stellar_core_commander/local_process.rb]: yes, used in `docker_process.rb`

**Pull request merged**